task/03-date-tasks.js

```diff
@@ -94,7 +98,15 @@ function timeSpanToString(startDate, endDate) {
  *    Date.UTC(2016,3,5,21, 0) => Math.PI/2
  */
 function angleBetweenClockHands(date) {
-    throw new Error('Not implemented');
+   let hours = date.getUTCHours();
+   let minutes = date.getUTCMinutes();
+   let angle = 0.5 * (60* hours + minutes) - 0.5 * 12 * minutes;
```

Comment:
A lot of magic numbers. What do they mean?

Reply:
these are the constants in the formula